pyproject.toml

```diff
@@ -114,6 +114,7 @@ all_extras = [
   'tsfresh>=0.17; python_version < "3.12"',
   'tslearn<0.7.0,!=0.6.0,>=0.5.2; python_version < "3.11"',
   "xarray",
+  "u8darts>=0.29.0",
```

Comment:
Please add an upper bound as well, e.g. `u8darts<0.30,>=0.29.0`. Same for below.

Reply:
Yeah, let me add bounds.
Also, do I need to add `darts` to `dl`

Reply:
Closing this particular thread, but my question to @fkiraly or @benHeid remains same. Darts in general has torch based .odels support, e.g. TFT from pytorch forecasting. Whether to add this is dl or forecasting needs to be decided before we merge this.

Reply:
@fkiraly I saw your note in HackMD. It's not resolved actually, I closed the thread as Github marked it outdated.

I think the question is still valid, or at least for me.

Reply:
I am currently exploring how `darts` handle soft dependencies. Once I'm done I will share a summary.